libgalois/src/PropertyGraph.cpp

```diff
@@ -132,11 +132,8 @@ WriteTopology(const katana::GraphTopology& topology) {
 katana::Result<std::unique_ptr<katana::PropertyGraph>>
 MakePropertyGraph(
     std::unique_ptr<tsuba::RDGFile> rdg_file,
-    std::optional<uint32_t> host_to_load,
-    const std::vector<std::string>* node_properties,
-    const std::vector<std::string>* edge_properties) {
-  auto rdg_result = tsuba::RDG::Make(
-      *rdg_file, host_to_load, node_properties, edge_properties);
+    const struct tsuba::RDGLoadOptions& opts) {
```

Comment:
you can take the C programmer out of .....

I think you meant `const tsuba::RDGLoadOptions& opts`

and probably drop `struct` throughout this PR.